### ISSUE TITLE
Screen readers fail to report the items in the list of added audio/video filters

### ISSUE THREAD

**Issue by k-kolev1985** [obsproject/obs-studio#3361]
### Description
This is an accessibility bug with OBS Studio. Screen readers like NVDA, Narrator and JAWS fail to read the items in the list of added audio/video filters. The focus is in the list, pressing up/down arrow moves between the added filters, but screen readers do not read the name of the selected filter and if it is enabled or disabled.

Can the problem be fixed? The problem was partially fixed some time ago for the "Sources" list in the main OBS Studio window. I say "partially", because screen readers now report the name of each source, but not if it is enabled or not (but the last one should go into another report, probably).

### Steps to reproduce it:
1. Launch a screen reader like NVDA, Narrator or JAWS.
2. Launch OBS Studio.
3. Create a scene and add at least one source to it.
4. From the context menu for the added source, select "Filters".
5. In the "Filters" dialog/window, add a few filters.
6. Move the focus to the list of filters and use up/down arrow to move between the filters in the list.

### Expected result:
The screen reader should report the filter's name or if it is enabled/disabled.

### Actual result.
The screen reader does not report the filter's name or if it is enabled/disabled.

### Test environment
- Operating system: Windows 10 Pro version 2004 (build 19041.450), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- OBS Studio version: 25.0.8 (64 bit), portable, in english.
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

**Comment by Fenrirthviti** [comment]
Closing as stale. If this is still an issue please open a new issue with updated reproduction steps on the latest version of OBS.

We have had several updates to Qt since this was reported, and this may have been resolved.
